### ISSUE TITLE
Links that are empty elements showing up as clickable links in authority records

### ISSUE THREAD

**Issue by AlisonBabeu** [agile-humanities/perseus_catalog#2]
For example, the authority record for [Eucrates Historicus](http://174.138.78.35:8080/exist/apps/PerseusCatalog/authors.html?id=urn:cite:perseus:author.1839.1)  lists five links, but only the two for VIAF and Smith work.

![image](https://user-images.githubusercontent.com/4663626/35813154-e4402796-0a60-11e8-9b19-d7a89edeb2d9.png)

So I check out the [authority record](https://github.com/PerseusDL/catalog_data/blob/master/mads/PrimaryAuthors/E/Eucrates_Historicus/author.1839.1.mads.xml) in Github 
```xml
  <mads:url displayLabel="Wikipedia"/>
  <mads:url displayLabel="Worldcat Identities"/>
  <mads:url displayLabel="Smith's Dictionary">
        http://www.perseus.tufts.edu/hopper/text?doc=Perseus:text:1999.04.0104:entry=eucrates-bio-3</mads:url>
  <mads:url displayLabel="VIAF">http://viaf.org/viaf/61941414</mads:url>
<mads:url displayLabel="id.gov"/>
```
So I'm guessing all of the URLs that are empty elements will need to be filtered out so they don't display?

**Comment by cwulfman** [comment]
Yes -- these pesky empty urls! But I can make a work-around, too.
